targets/mate

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh -e
+# Copyright (c) 2016 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+if [ "${TARGETNOINSTALL:-c}" = 'c' ]
+   if [ "${ARCH#arm}" != "$ARCH" ]; then
+       error 99 "Mate packages are not available for ARM."
```

Comment:
Ooh, sorry, I should have realized earlier that Mate isn't available for ARM.
Historically I haven't merged DEs unless they're available on a vast majority of Chromebooks (there's quite a lot of ARM) and on a majority of the supported releases.

I think Mate now squeaks by on the latter point, but I'm extremely hesitant to include something that only works on certain architectures.

@drinkcat @DennisLfromGA thoughts?